src/test/java/codeu/model/store/basic/IndexStoreTest.java

```diff
@@ -23,20 +23,40 @@
 
   private final UUID CONVERSATION_ID_ONE = UUID.randomUUID();
   private final Message MESSAGE_ONE =
-      new Message(
-          UUID.randomUUID(),
-          CONVERSATION_ID_ONE,
-          UUID.randomUUID(),
-          "This is test message one.",
-          Instant.ofEpochMilli(1000));
-  
+        new Message(
+            UUID.randomUUID(),
+            CONVERSATION_ID_ONE,
+            UUID.randomUUID(),
+            "This is test message one.",
+            Instant.ofEpochMilli(1000));
+    private final Message MESSAGE_TWO =
+        new Message(
+            UUID.randomUUID(),
+            CONVERSATION_ID_ONE,
+            UUID.randomUUID(),
+            "This is test message two.",
+            Instant.ofEpochMilli(2000));
+    private final Message MESSAGE_THREE =
+        new Message(
+            UUID.randomUUID(),
+            CONVERSATION_ID_ONE,
+            UUID.randomUUID(),
+            "This is test message three.",
+            Instant.ofEpochMilli(3000));
+    private final Message MESSAGE_FOUR =
+        new Message(
+            UUID.randomUUID(),
+            CONVERSATION_ID_ONE,
+            UUID.randomUUID(),
+            "is Random test message one four.",
+            Instant.ofEpochMilli(4000));
```

Comment:
Could you add one more boolean value to every Message object when you call the object? I added one more field to message class :)). For example: 
 private final Message MESSAGE_FOUR =
+        new Message(
+            UUID.randomUUID(),
+            CONVERSATION_ID_ONE,
+            UUID.randomUUID(),
+            "is Random test message one four.",
+            Instant.ofEpochMilli(4000), false);